### ISSUE TITLE
model metrics visualization key error

### ISSUE THREAD

**Issue by amarlearning** [keras-team/keras#13624]
In the given [documentation](https://keras.io/visualization/), the mentioned key are `acc` and `val_acc`, but actually it is `accuracy` and `val_accuracy`.

Given documentation screenshot:
![Screenshot from 2019-12-11 15-05-42](https://user-images.githubusercontent.com/9383897/70609796-70811a00-1c28-11ea-9ff5-29fcad5b2065.png)

Whereas the actual keys are `dict_keys(['val_loss', 'val_accuracy', 'loss', 'accuracy'])`

Changes to be made in [this](https://github.com/keras-team/keras/blob/master/docs/templates/visualization.md) file.

**Comment by am1tyadav** [comment]
Does it not depend on the metrics set when compiling a model? For example, if you compile a model like this:
```
model.compile(
  loss="binary_crossentropy",
  optimizer="adam",
  metrics=["acc"]
```
Then the history object will have keys `["loss", "val_loss", "acc", "val_acc"]`

But if you compiled like:
```
model.compile(
  loss="binary_crossentropy",
  optimizer="adam",
  metrics=["accuracy"]
```
Then the history object will have keys `["loss", "val_loss", "accuracy", "val_accuracy"]`

**Comment by amarlearning** [comment]
@am1tyadav well then in that case it should'nt be dependent on the key that we pass in metrics while compiling the model.

I think the keys that we get from history object should be consistent everytime.

**Comment by AnirbanMukherjeeXD** [comment]
In [this notebook](https://github.com/AnirbanMukherjeeXD/cnn-mnist-digits/blob/master/ConvNet_MNIST.ipynb) I've implemented CNN using Keras. 
I have used  `metrics=['accuracy']` and on printing `print(history.history.keys())` it shows 
`dict_keys(['val_loss', 'val_acc', 'loss', 'acc'])`



![s](https://user-images.githubusercontent.com/34798203/72805272-52cfa900-3c07-11ea-95db-c39cfcbf54cb.png)

**Comment by am1tyadav** [comment]
> In [this notebook](https://github.com/AnirbanMukherjeeXD/cnn-mnist-digits/blob/master/ConvNet_MNIST.ipynb) I've implemented CNN using Keras.
> I have used `metrics=['accuracy']` and on printing `print(history.history.keys())` it shows
> `dict_keys(['val_loss', 'val_acc', 'loss', 'acc'])`
> 
> ![s](https://user-images.githubusercontent.com/34798203/72805272-52cfa900-3c07-11ea-95db-c39cfcbf54cb.png)

Yeah, that seems to be the case in tf.keras, not in keras
